pkg/cidata/user-data.TEMPLATE

```diff
@@ -63,11 +186,16 @@ write_files:
       umount /mnt/lima-cidata
 
       # Launch the guestagent service
-      until [ -e "/run/user/{{.UID}}/systemd/private" ]; do sleep 3; done
-      sudo -iu "{{.User}}" "XDG_RUNTIME_DIR=/run/user/{{.UID}}" lima-guestagent install-systemd
+      if [ -f /etc/alpine-release ]; then
+        rc-update add lima-guestagent default
+        rc-service lima-guestagent start
```

Comment:
On systemd host, we have `lima-guestagent install-systemd`, so maybe we should have `lima-guestagent install-openrc`?

Reply:
I actually started that way, and may even still have the branch here locally, but then realized that including the file in `user-data` is much easier.

Added complexity is that the OpenRC script must be installed by `root`, and `lima-guestagent` doesn't allow running as `root`, so that logic also became more complicated.

Is there a reason not to generate the `systemd` config via template in `user-data` (or as a separate file) as well?

Reply:
I think we can modify the guest agent to allow running as the root.

> Is there a reason not to generate the systemd config via template in user-data (or as a separate file) as well?

No specific reason

Reply:
> > Is there a reason not to generate the systemd config via template in user-data (or as a separate file) as well?
> 
> No specific reason

In that case I would want to simplify the systemd config in a separate commit, once the user-data refactoring is done.